### PR TITLE
[Continued] Implemented Jacchia77 atmospheric model 

### PR DESCRIPTION
**Pull request by DhruvSondhi**
Hello, 
This PR aims to continue the effort made to implement Jacchia77 atmospheric model (Thanks to @iamabhishek0 for the work)
Cheers :)

### CODE REVIEW

**Review by astrojuanlu — CHANGES_REQUESTED** [review]
Thanks a lot for bringing this back to life @DhruvSondhi ! Please address the review comments that @jorgepiloto did on #944 and I think we are ready to go

**Comment by astrojuanlu** [comment]
Also, this will need a rebase

**Comment by DhruvSondhi** [comment]
Hello @astrojuanlu, I have done the requested changes 😄 ... Though we need to implement the `variation due to geomagnetic activity` & `diurnal variations in the atmosphere density` ... We can merge this into the library but there are some changes that need to be done like the Density computation here has a very weird unit & we need to address it ... I have found much legible material for implementation but I need some confirmation about the same ... So, Yes we have some work to do 😃

File: tests/tests_atmosphere/test_jacchia77.py [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+    p = np.log10(pressure.value) * pressure.unit
+
+    assert_quantity_allclose(p, expected_p, rtol=1e-3)
+
+
+@pytest.mark.parametrize("z", jacchia77_solutions.keys())
+def test_density(z):
+    expected_rho = jacchia77_solutions[z][10] * (
+        u.kg ** 2 * u.mol * u.K ** -1 * u.m ** -3
+    )
```
Comment (DhruvSondhi) [inline tests/tests_atmosphere/test_jacchia77.py:263]: @astrojuanlu Check this out ... I tried fixing this but got confused ... Will need to spend some time checking where the implementation has gone wrong as these are clearly not the units for `Density` 😞

**Review by DhruvSondhi — COMMENTED** [review]
Checking for a bug

**Comment by DhruvSondhi** [comment]
@jorgepiloto Can you check the implementation? I have made the requested changes but need to discuss the implementation as it seems to be specific implementation than a more general one (Though I am not sure about it 🤔 )

**Review by astrojuanlu — APPROVED** [review]
This is good enough to be merged, and we can address any bugs or concerns afterwards 👍🏽 Thanks a lot @DhruvSondhi!

**Comment by DhruvSondhi** [comment]
Sure, Would be happy to contribute & add the remaining Atmospheric Model 😄

**Pull request merged**